coverage_ai/lsp_logic/multilspy/language_server.py

```
@@ -425,10 +425,7 @@ async def request_definition(
                     new_item["range"] = item[LSPConstants.TARGET_SELECTION_RANGE]
                     ret.append(multilspy_types.Location(**new_item))
                 else:
-                    self.logger.log(
-                        f"Unexpected response from Language Server: {item}",
-                        logging.ERROR,
-                    )
+                    assert False, f"Unexpected response from Language Server: {item}"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)